extensions/data-workspace/src/common/constants.ts

```diff
@@ -40,6 +40,7 @@ export const ProjectParentDirectoryNotExistError = (location: string): string =>
 export const ProjectDirectoryAlreadyExistError = (projectName: string, location: string): string => { return localize('dataworkspace.projectDirectoryAlreadyExistError', "There is already a directory named '{0}' in the selected location: '{1}'.", projectName, location); };
 export const WorkspaceFileInvalidError = (workspace: string): string => { return localize('dataworkspace.workspaceFileInvalidError', "The selected workspace file path '{0}' does not have the required file extension {1}", workspace, WorkspaceFileExtension); };
 export const WorkspaceParentDirectoryNotExistError = (location: string): string => { return localize('dataworkspace.workspaceParentDirectoryNotExistError', "The selected workspace location '{0}' does not exist or is not a directory", location); };
+export const WorkspaceFileAlreadyExistsError = (file: string): string => { return localize('dataworkspace.workspaceFileAlreadyExistsError', "The selected workspace file '{0} already exists. To add the project to an exising workspace, use the Open Existing dialog to first open the workspace", file); };
```

Comment:
Typos:
* to an exis*t*ing workspace
* missing closing single-quotation mark around `{0}`